src/provider.ts

```diff
@@ -268,7 +272,6 @@ export class ProviderLedgerReactNative
           case '27010':
             this.emit(source, false, e.message, e.name, '27010');
             throw new Error('ledger_locked');
-            break;
```

Comment:
Why was the removed break?

Reply:
We have throw new Error on previous line, so this break is unused
(according to logic and eslint :))